modules/imgcodecs/src/grfmt_png.cpp

```diff
@@ -1416,8 +1418,9 @@ void PngEncoder::deflateRectFin(unsigned char* zbuf, uint32_t* zsize, int bpp, i
 
 bool PngEncoder::writeanimation(const Animation& animation, const std::vector<int>& params)
 {
-    int compression_level = 6;
-    int compression_strategy = IMWRITE_PNG_STRATEGY_RLE; // Default strategy
+    if (animation.frames.size() == 1)
```

Comment:
I still think this is an error: a 1-frame APNG is not the same as a PNG. The spec allows for 1 frame APNG: "1 is a valid value for a single-frame APNG.", cf https://wiki.mozilla.org/APNG_Specification

Reply:
@vrabaud i removed this condition. but fyi 1 frame apng still saved without acTL and fcTL

Reply:
Well, that is a bug in itself. Sure, usually, when you ask for a 1-frame APNG, the user probably wants a PNG but OpenCV should not be the one to decide.